.dockerignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
lol, not that it matters, but we'll never have a `node_modules` in here anyway right